azure-iot-device/tests/common/transport/mqtt/test_pipeline_events_mqtt.py

```diff
@@ -0,0 +1,20 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+from azure.iot.device.common.transport.mqtt import pipeline_events_mqtt
+
+fake_topic = "__fake_topic__"
+fake_payload = "__fake_payload__"
+
+
+@pytest.mark.describe("IncomingMessage object")
+class TestIncomingMessage(object):
+    @pytest.mark.it("Sets arguments correctly")
```

Comment:
>("Sets arguments correctly") [](start = 19, length = 28)

This isn't really a specific requirement, in a doc requirement style. What does "sets arguments correctly" mean?

I recommend breaking things like this up into logical groups that translate to actual requirements.

i.e.

1) "Sets polymorphic class name" (is that the right term for this pattern where the name is in the class? I can't remember)
2) "Sets topic and payload from message"